pyecharts/translator/api.py

```diff
@@ -0,0 +1,135 @@
+# coding=utf8
+
+"""
+The api module for python-to-javascript translator
+"""
+
+from __future__ import unicode_literals
```

Comment:
not required unless you have Chinese characters in the file.

Reply:
Use `unicode_literals` to keep consistent in using **text** string in Python 2 and 3. , although most function accept both **text** and **binary** as parameter.